capsules/extra/src/atecc508a.rs

```diff
@@ -12,11 +12,11 @@
 //! must be pulled low.
 //!
 //! The ATECC508A is shipped in an unlocked state. That is, the configuration
-//! can be changed. The ATECC508A is practically usless while it's unlocked
+//! can be changed. The ATECC508A is practically useless while it's unlocked
 //! though. Even the random number generator only returns
 //! 0xFF, 0xFF, 0x00, 0x00 when the device is unlocked.
 //!
-//! Locking the device is permanant! Once the device is locked it can not be
+//! Locking the device is permanent! Once the device is locked it can not be
```

Comment:
The last phrase of this comment does not seem quite right, but my English understanding might be wrong :)

Reply:
Just a spelling fix